### PR TITLE
Jasmine client options

### PR DESCRIPTION
**Pull request by tilsammans**
Continues #519 

Thanks to everyone involved!

### CODE REVIEW

File: lib/teaspoon/driver/selenium.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def initialize(options = nil)
       end
 
       def run_specs(runner, url)
-        driver = ::Selenium::WebDriver.for(driver_options[:client_driver])
+        driver = ::Selenium::WebDriver.for(driver_options[:client_driver], @options[:client_driver_opts]&.dup || {})
```
Comment (zzak) [inline lib/teaspoon/driver/selenium.rb]: Doesn't this also make a dependency on whatever Ruby version introduced this new syntax? I think 2.3?

**Comment by jsilvestri** [comment]
`options[:client_driver_opts].symbolize_keys` works as well and might address the syntax concerns here: https://github.com/jsilvestri/teaspoon/commit/7ec4b6d6e599cfe1a446ceea84a2e674550b0ca6

**Comment by zzak** [comment]
@jsilvestri I believe `Hash#symbolize_keys` is only available in ActiveSupport, but not sure that is a concern here.

**Comment by jsilvestri** [comment]
@zzak correct.  Agreed it may not be ideal to depend on Active Support.

**Comment by mathieujobin** [comment]
looks like #537 properly fix this issue, please reopen if you disagree

**Pull request closed without merging**